### PR TITLE
Use carrierwave-aws instead of fog

### PR DESCRIPTION
**Pull request by josepjaume**
#### :tophat: What? Why?
This uses carrierwave-aws instead of fog in order to prevent  unnecessary `HEAD` requests to s3.

#### :pushpin: Related Issues
Someone complained about this on carrierwave/fog but didn't get an answer:
https://github.com/carrierwaveuploader/carrierwave/issues/1655

#### :clipboard: Subtasks
*None*

### :camera: Screenshots (optional)
*None*

### CODE REVIEW

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -6,23 +6,25 @@
...
+    config.aws_acl    = 'public-read'
+    config.aws_authenticated_url_expiration = 60 * 60 * 24 * 7
+
+    config.aws_attributes = {
+      expires: 1.week.from_now.httpdate,
+      cache_control: 'max-age=604800'
+    }
+
+    config.aws_credentials = {
+      access_key_id:     Rails.application.secrets.aws_access_key_id
```
Comment (mrcasals) [inline config/initializers/carrierwave.rb]: There's a comma missing here

**Comment by mrcasals** [comment]
@josepjaume I don't understand the problem, I don't have context here. Care to explain?

**Comment by mrcasals** [comment]
TLDR: there's a `HEAD~ request every time we load a page that uses an uploader. Replacing `fog` with `carrierwave-aws` removes this unnecessary request.

**Comment by josepjaume** [comment]
@mrcasals allegedly, will see.
